docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -19,54 +19,42 @@ As such, it can be helpful to scan a monorepo in parts for multiple reasons:
 
 When scanning a repo with Semgrep in CI, the base command is `semgrep ci`. To understand this default setup for your source code manager (SCM) and CI provider, see [Getting started with Semgrep in continuous integration (CI)](/deployment/add-semgrep-to-ci).
 
-To split up your monorepo, you need to make two changes. First, use the `--include` flag to determine *how* you want to logically split up the code. Second, update the `SEMGREP_REPO_DISPLAY_NAME` environment variable to assign findings to separate projects in Semgrep AppSec Platform. 
+There are two tools provided to split up a repo. Consider a monorepo named `monorepo` with four main modules
```

Comment:
```suggestion
There are two features provided by Semgrep to split up a repo. Consider a monorepo named `monorepo` with four main modules
```

Does this refer to the --subdir flag and the SEMGREP_REPO_DISPLAY_NAMES env variable?

Reply:
It's referring to `--subdir` and the `--include + SEMGREP_REPO_DISPLAY_NAME` combination!